src/main/java/graphql/ExecutionResult.java

```diff
@@ -11,6 +11,13 @@
 
     List<GraphQLError> getErrors();
 
-    Map<Object,Object> getExtensions();
+    Map<Object, Object> getExtensions();
```

Comment:
@andimarek - the spec says:

> The

 response map may also contain an entry with key extensions. This entry, if set, must have a map as its value. This entry is reserved for implementors to extend the protocol however they see fit, and hence there are no additional restrictions on its contents.

So since graphql-java (the implementor) does not use it then I would remove it until some one comes up with a good use case for it.

And perhaps we should have  a ExecutionResultBuilder so that it ExecutionResult can be immutable more easily